src/NetAppFiles/NetAppFiles/Account/NewNetAppFilesAccount.cs

```diff
@@ -46,25 +45,32 @@ public class NewAzureRmNetAppFilesAccount : AzureNetAppFilesCmdletBase
         [LocationCompleter("Microsoft.NetApp/netAppAccounts")]
         public string Location { get; set; }
 
-        [Parameter(
-            Mandatory = false,
-            HelpMessage = "A hashtable which represents resource tags")]
-        [ValidateNotNullOrEmpty]
-        [Alias("Tags")]
-        public Hashtable Tag { get; set; }
-
         [Parameter(
             Mandatory = true,
             HelpMessage = "The name of the ANF account")]
         [ValidateNotNullOrEmpty]
         [Alias("AccountName")]
         public string Name { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A hashtable array which represents the active directories")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable[] ActiveDirectories { get; set; }
```

Comment:
What are the contets of this hashtable?  Also, parameter names should be singular 'ActiveDirectoryConfiguration' for example.